first-bot/adding-a-config-file.md

```diff
@@ -61,10 +61,10 @@ const prefix = "!";
 client.on("message", (message) => {
```

Comment:
```suggestion
client.on("messageCreate", (message) => {
```

Reply:
Fixed in b5c8ebfc63f03855f21e383c7d0c06ddeea78255